components/apimgt/org.wso2.carbon.apimgt.tracing/src/main/java/org/wso2/carbon/apimgt/tracing/TracingConstants.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.apimgt.tracing;
+
+class TracingConstants {
+
+    /**
+     * OpenTracer Constants
+     * */
+    static final String OPEN_TRACER_NAME = "OpenTracer.Name";
+    static final String DEFAULT_OPEN_TRACER_NAME = "zipkin";
+    static final String OPEN_TRACER_ENABLED = "OpenTracer.Enabled";
+    static final String DEFAULT_OPEN_TRACER_ENABLED = "true";
```

Comment:
Let's have the tracer enabled false by default. Because the customers should enable this by knowing that they are doing this.